.circleci/config.yml

```diff
@@ -1360,41 +1360,95 @@ workflows:
           requires:
             - 🚧-🌿-<< matrix.host >>-🕹️
 
-      # Deploy to pre-prod (for release branches)
+      # Bump Tags (for release branch hot-fixes)
       - noop:
-          name: 🚧-🎁-to-pre-prod-🕹️
-          type: approval
+          name: bump-hotfix-tags
           filters:
             branches:
               only: /(^release-.*$)/
           requires:
             - built
+      # Deploy Discovery Nodes to prod (for release branches)
+      - noop:
+          name: 🚧-🎁-to-🔍-prod-🕹️
+          type: approval
+          filters:
+            branches:
+              only: /(^release-.*$)/
+          requires:
+            - bump-hotfix-tags
       - deploy:
-          name: 🚧-🎁-to-pre-prod
+          name: 🚧-🎁-to-🔍-prod
           git_tag: ${CIRCLE_SHA1}
-          environment: pre-prod
-          service: all
+          environment: prod
+          service: discovery
+          context:
+            - slack-secrets
+            - gh-tokens
+            - open-vpn
+          requires:
+            - 🚧-🎁-to-🔍-prod-🕹️
+      # Deploy Identity to prod (for release branches)
+      - noop:
+          name: 🚧-🎁-to-🆔🪪-prod-🕹️
```

Comment:
I think the emojis are maybe getting a bit hard understand

maybe sth like?
🚧-release-is-prod-🕹️
🚧-release-dn-prod-🕹️
deploy-release-cn-prod-🕹️

Reply:
no emojis would be easier to understand

Reply:
the reason for the emojis is circleci truncates characters so they all say the same thing :/

Reply:
but i agree. just not sure how to solve.

Reply:
Indeed, I like your suggestion of is, cn, dn! Implemented.

Do let me know if either of you have more ideas. I continue to be open to all naming suggestions as we use emojis as a stop gap to building the rest of the deployment pipeline's functionality.